src/sass/layouts/hero.scss

```diff
@@ -0,0 +1,416 @@
+html {
+    box-sizing: border-box;
+  }
+  
+  *,
+  *::after,
+  *::before {
+    box-sizing: inherit;
+  }
+  
+  ul,
+  ol {
+    padding: 0;
+  }
+  
+  body,
+  h1,
+  h2,
+  h3,
+  h4,
+  h5,
+  h6,
+  p,
+  ul,
+  ol,
+  li,
+  figure,
+  figcaption,
+  blockquote,
+  dl,
+  dd {
+    margin: 0;
+  }
+    
+    li {
+      list-style: none;
+    }
+  .container{
+    margin: 0 auto;
+    padding-left: 15px;
+    padding-top: 15px;
+  }
+  
+  
+  //   body{
+  // background-color: #202025;
+  
+  //   }
+  .is-hiden{
+    display: none;
+   }
+  
+  .hero{
+   
+  &-title{
+      font-family: 'Poppins';
+      font-style: normal;
+    
+      color: $second-text-color;
+    }
+    &-text{
+      display: block;
+      font-family: 'Poppins';
+      font-style: normal;
+      color: $dark-primary-color;
+    }
+   &-list{
+      justify-content: center;
+    }
+    &-button{
+      border: 0;
+      background: transparent;
+      justify-content: center;
+      font-family: 'Poppins';
+      font-style: normal;
+      align-items: center;
+      color: $second-text-color;
+      
+    }
+  }
+  .coctails-section-coctailTitel{
+    margin: 0 auto;
+    font-family: 'Poppins';
+font-style: normal;
+font-weight: 700;
+align-items: center;
+text-align: center;
+    display: flex;
+   justify-content: center;
```

Comment:
TODO: 
1) lines 86-88 replace with the @mixin flex (center)
2) line 83 replace with var '$main-font'